getting-started/local-node/setting-up-a-node.md

```diff
@@ -182,4 +207,4 @@ bottom drive obey lake curtain smoke basket hold race lonely fit walk
 
 Checkout the [Using MetaMask](/getting-started/local-node/using-metamask/) section to get started interacting with your accounts.
 
---8<-- 'text/setting-up-local/dev-accounts.md'
+--8<-- 'text/setting-up-node/dev-accounts.md'
```

Comment:
same as above... for the text snippets, it looks like the directory is still "setting-up-local" instead of "setting-up-node" so this snippet is not rendering on the page